### ISSUE TITLE
Fix Tab Orders

### ISSUE THREAD

**Issue by JACorley1** [JACorley1/UWG-SE1-TimeManagementSystem#1]
The tab orders for the UI do not flow naturally. These should be corrected to bring the user naturally through the interface as they are working.
